src/main/java/com/boot/bankservice/service/impl/TransactionServiceImpl.java

```diff
@@ -37,7 +37,7 @@ public Transaction completeTransaction(String accountNumberFrom,
             throw new DataProcessingException("Balance can't be less than 0 ");
         }
         transactionRepository.save(outcome);
-        accountService.add(accountFrom);
+        accountService.save(accountFrom);
```

Comment:
if you insist on leaving `update` method, won't it be more suitable here?

Reply:
agree